provisioning/local/env-scripts/extract-variables-common.sh

```diff
@@ -5,15 +5,24 @@ cd "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source ./functions.sh
 
 # output variables
-output_var 'TEST_TENANT_ID' $(terraform_output 'az_tenant_id')
-output_var 'TEST_CLIENT_ID' $(terraform_output 'az_application_id')
-output_var 'TEST_CLIENT_SECRET' $(terraform_output 'az_application_secret')
-output_var 'TEST_KEY_VAULT_URL' $(terraform_output 'az_key_vault_url')
+output_var 'TEST_TENANT_ID' "$(terraform_output 'az_tenant_id')"
+output_var 'TEST_CLIENT_ID' "$(terraform_output 'az_application_id')"
+output_var 'TEST_CLIENT_SECRET' "$(terraform_output 'az_application_secret')"
+output_var 'TEST_KEY_VAULT_URL' "$(terraform_output 'az_key_vault_url')"
 echo ""
 
-output_var 'TEST_AWS_ACCESS_KEY_ID' $(terraform_output 'aws_access_key_id')
-output_var 'TEST_AWS_SECRET_ACCESS_KEY' $(terraform_output 'aws_access_key_secret')
-output_var 'TEST_AWS_REGION' $(terraform_output 'aws_region')
-output_var 'TEST_AWS_KMS_KEY_ID' $(terraform_output 'aws_kms_key_id')
-output_var 'TEST_AWS_ROLE_ID' $(terraform_output 'aws_role_arn')
+output_var 'TEST_AWS_ACCESS_KEY_ID' "$(terraform_output 'aws_access_key_id')"
+output_var 'TEST_AWS_SECRET_ACCESS_KEY' "$(terraform_output 'aws_access_key_secret')"
+output_var 'TEST_AWS_REGION' "$(terraform_output 'aws_region')"
+output_var 'TEST_AWS_KMS_KEY_ID' "$(terraform_output 'aws_kms_key_id')"
+output_var 'TEST_AWS_ROLE_ID' "$(terraform_output 'aws_role_arn')"
+
+output_var 'TEST_GOOGLE_CLOUD_PROJECT' "$(terraform_output 'gcp_project_id')"
+output_var 'TEST_GCP_KMS_KEY_ID' "$(terraform_output 'gcp_kms_key_id')"
```

Comment:
`GOOGLE` vs `GCP` - nemohli bysme sjednotit a prejmenovat treba na `TEST_GCP_PROJECT_ID`? Nebo to SDK nejak automaticky konzumuje `GOOGLE_CLOUD_PROJECT` ENV?

Reply:
Btw. je potreba to separatne nastavovat? Neni to v tom service account klici?

Reply:
no GOOGLE_APPLICATION_CREDENTIALS  je název danej https://cloud.google.com/docs/authentication/application-default-credentials#GAC, jako stejně tak je tam TEST_TENANT_ID no

jo, ten projekt v outputu potřeba není, to mi tam zůstalo z pokusu

Reply:
jo, `GOOGLE_APPLICATION_CREDENTIALS` znam, `GOOGLE_CLOUD_PROJECT` mi ale prave nic nerikalo, tak me to zajimalo.. kdyz to jde pryc, tak tim lip :slightly_smiling_face: